resources/application-connector/charts/application-broker/templates/cluster-role.yaml

```diff
@@ -29,4 +30,11 @@ rules:
 - apiGroups: ["servicecatalog.k8s.io"]
   resources: ["servicebrokers"]
   verbs: ["get", "create", "delete", "list", "update"]
-
+# subscriptions are created by the application-broker to enable events flowing from a channel to the broker
+- apiGroups: ["messaging.knative.dev"]
+  resources: ["subscriptions"]
+  verbs: ["get", "create", "delete", "list", "update"]
+# readonly access to channels is required to create a subscription (channel is part of subscription spec)
+- apiGroups: ["messaging.knative.dev"]
+  resources: ["channels"]
+  verbs: ["get", "list"]
```

Comment:
Same here, `get` doesn't seem to be used anywhere in the broker.

Reply:
Fixed